src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -492,11 +492,13 @@
                 (.writeObject w (.-node-id ^RuleOrderedActivation c) true)
```

Comment:
Could we recap the discussion about the non-usage of this handler in a sentence or two of comments inline here?